### PR TITLE
fix: release variant of storybooknative app

### PR DESCRIPTION
**Pull request by rjanjua**
requires dextrose snapshots to pass in bitrise, manually running the CI jobs as we disabled PRs. 

The fix involved 
- swapping babel-preset-env for babel-preset-react-native-stage-0 
- adding a script in the iOS build phases to copy across the js bundle for a release variant only

### CODE REVIEW

**Comment by kaliabadi** [comment]
just waiting for bitrise then happy to merge

**Comment by rjanjua** [comment]
@nikhedonia looks like it's because of the `article-labels` story's `addon/knobs` dependencies doing unexpected stuff

**Comment by rjanjua** [comment]
closed and reopened to trigger PR

**Comment by rjanjua** [comment]
ios ci build fails for an unrelated reason to this PR - as far as this PR is concerned, the release variant is fixed

shots: https://s3-eu-west-1.amazonaws.com/times-components-snaps/c790d6f8d107eb1d33108fb897abdb707558c8e2/dextrose-test/index.html

**Comment by nikhedonia** [comment]
I can confirm it works when knobs are removed from the codebase

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/c790d6f8d107eb1d33108fb897abdb707558c8e2/dextrose-test/index.html

**Comment by nikhedonia** [comment]
could you check whether it is possible to use `stage-2` or `stage-1` ?

File: package.json [hunk 1]
```diff
@@ -147,7 +148,6 @@
     "webpack": "3.3.0"
   },
   "dependencies": {
-    "babel-preset-env": "1.6.1",
```
Comment (craigbilner) [inline package.json]: this is a peer dep for every package

**Comment by rjanjua** [comment]
https://s3-eu-west-1.amazonaws.com/times-components-snaps/b11f2f5c4bd10fc7ed70e743a8fec9d2eb109de4/dextrose-test/index.html 

Me and Kouros tested storybooks too, they look fine. Though if @craigbilner or @nikhedonia could check on your linux/android machines would be useful 👍

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/2190e7cc2e620caf6b2897aaddacf7bef23d702b/dextrose-test/index.html

**Pull request merged**